Druid-Tests/DRPrimitiveScenarioCompilationTest.class.st

```diff
@@ -1999,6 +1999,16 @@ DRPrimitiveScenarioCompilationTest >> testPrimitiveWithDeadCodeGoesThroughSecond
 	self assert: machineSimulator receiverRegisterValue equals: 55
 ]
 
+{ #category : #'tests-comparisons' }
+DRPrimitiveScenarioCompilationTest >> testRotateLeft [
+
+	self compileDruidPrimitive: #primitiveRotateLeft.
+	
+	self executePrimitiveWithReceiver: 42.
+	
+	self assert: machineSimulator receiverRegisterValue equals: 42 << 1
+]
```

Comment:
Would be nice to add a case where the rotate doesn't have the same results as a shift.